pkg/distro/BUILD

```diff
@@ -40,6 +40,7 @@ pkg_tar(
         "//private:standard_package",
         "//releasing:standard_package",
         "//toolchains:standard_package",
+        "//toolchains/git:standard_package",
```

Comment:
I guess so.  But I am actually dubious about the whole thing.
The hoops I jump through to get bazel to run git in the source tree instead of a sandbox are a little strange. OTOH, even if I toss out the toolchain and change it to provide tools that run git manually, standard_package would still be part of it.